src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -114,4 +123,5 @@
       </div>
     </div>
   </div>
```

Comment:
Can you reindent https://github.com/PrestaShop/PrestaShop/pull/21923/files/be6f812114b5350b06166b47bb73d33628ce40a9..a705879047ee8241d215a1b427b81cdb43b47c43#diff-91bfb53e98337d53ca1c3df53d73004fba4a420f24da57ea9fc3740b498f2d80R109-R125 ?

Reply:
Done, if I understood this correctly.